run_e2e.sh

```diff
@@ -42,13 +42,17 @@ ginkgo_args=(
     "--noColor=true"
 )
 
+# Attempt to detect the number of nodes in the cluster on which workloads
+# may be scheduled.
+NUM_NODES="${NUM_NODES:-$(kubectl get nodes -o=jsonpath='{range .items[*]}{.spec.taints}{"\n"}{end}' | grep -cv 'effect:NoSchedule')}"
```

Comment:
This should really be an initialization function of the framework vs. something external.

Reply:
tracking issue is here: https://github.com/kubernetes/kubernetes/issues/74853

Reply:
Hi @timothysc,

I agree 100%, and that's essentially what BenTheElder on Slack was saying as well. However, I encountered the problem while trying to fix broken conformance tests, and I fixed what I could on my end.

If you're fine with conformance against `ci/latest` failing until `e2e.test` has a sane default (it's currently `-1`) for `-num-nodes`, then I'm fine letting them fail as well.